scout/commands/delete/delete_command.py

```diff
@@ -118,18 +120,22 @@ def case(institute, case_id, display_name):
                 "deleted with flag '-i/--institute'."
             )
             raise click.Abort()
+        case_obj = adapter.case(display_name=display_name, institute_id=institute)
+    else:
+        case_obj = adapter.case(case_id=case_id)
+
+    if not case_obj:
+        click.echo(
+            "Coudn't find any case in database matching the provided parameters."
+        )
+        raise click.Abort()
 
     LOG.info("Running deleting case {0}".format(case_id))
-    case = adapter.delete_case(
-        case_id=case_id, institute_id=institute, display_name=display_name
-    )
+    case = adapter.delete_case(case_id=case_obj["_id"])
 
     if case.deleted_count == 1:
```

Comment:
What if not `case.deleted_count == 1`? I think we should either remove this `if` or add a clause when it os false

Reply:
I don't think that it will ever be false, especially if a few lines before it has checked that the case with that id exists. I'd remove it